Add new files

Joey19900 opened the pull request.

Commit:
Add new files

diegonava6 commented on src/App.css:
```diff
@@ -1,38 +1,55 @@
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+/*contenedor*/
```
No comments in spanish, change all of them in all your files

diegonava6 commented on src/App.css:
```diff
@@ -1,38 +1,55 @@
+/*contenedor*/
+.task-list-main {
+  width: 30%;
+  margin: 20x;
```
You mean 
margin: 20px;

?

diegonava6 commented on src/App.js:
```diff
@@ -1,23 +1,16 @@
-        </a>
-      </header>
+    <div className="todoApp">
+      <div className="top-botton"></div>
```
ClassName is not clear, you have a typo

diegonava6 commented on src/App.js:
```diff
@@ -1,23 +1,16 @@
-import './App.css';
+import React from "react";
+import "./App.css";
+import taskForm from "./taskForm";
```
Name of components should be Capitalized:
TaskForm

diegonava6 commented on src/buttons.jsx:
```diff
@@ -0,0 +1,54 @@
+  const showAlert = () => {
+    swal({
+      title: "confirm",
+      text: "Are you sure you want to delete this task?",
```
warning instead of waring

diegonava6 replied:
It would be better to use 'Warning'

It looks weird for the user if this is in lowercase

diegonava6 replied:
It would be better to use 'Warning'

It looks weird for the user if this is in lowercase

diegonava6 replied:
It would be better to use 'Warning'

It looks weird for the user if this is in lowercase

diegonava6 commented on src/buttons.jsx:
```diff
@@ -0,0 +1,54 @@
+      title: "confirm",
+      text: "Are you sure you want to delete this task?",
+      icon: "waring",
+      button: ["acept", "cancel"],
```
accept instead of acept

diegonava6 commented on src/buttons.jsx:
```diff
@@ -0,0 +1,54 @@
+          minLength={3}
+          maxLength={30}
+          placeholder="write"
+          onChange={(e) => setTitle(e.target.value)}
```
Create a function instead of using an arrow function directly here

For example, in the form, you have inside the onSubmit event a function called “handleSubmit”

diegonava6 commented on src/buttons.jsx:
```diff
@@ -0,0 +1,54 @@
+        <textarea
+          maxLength={100}
+          placeholder="Describe your task"
+          onChange={(e) => setDescription(e.target.value)}
```
Same comment here as I mentioned on the onChange of input

diegonava6 reviewed (changes requested):
Check and correct the comments I left

Joey19900 commented:
Ok. Bro thanks

On Tue, Nov 1, 2022, 10:49 AM Diego Nava ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Check and correct the comments I left
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Joey19900/todoApp/pull/1#pullrequestreview-1163681099>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AY4HO3KB4NB33ADUXEJMMBLWGFCZJANCNFSM6AAAAAARUBNJBU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

diegonava6 commented on src/buttons.jsx:
```diff
@@ -0,0 +1,54 @@
+
+  const showAlert = () => {
+    swal({
+      title: "confirm",
```
It would be better to use 'Confirm'

It looks weird for the user if this is in lowercase

diegonava6 replied:
It would be better to use 'Confirm'

It looks weird for the user if this is in lowercase

diegonava6 replied:
It would be better to use 'Confirm'

It looks weird for the user if this is in lowercase

diegonava6 commented on src/buttons.jsx:
```diff
@@ -0,0 +1,54 @@
+    </div>
+  );
+}
+//mandar pullrequest antes del demo
```
Remove comment

No comments in spanish

diegonava6 replied:
Remove comment

No comments in spanish

diegonava6 commented on src/task.jsx:
```diff
@@ -0,0 +1,16 @@
+import React from "react";
```
The file name should be 'Task' not 'task'

The convention for components names is to be capitalized

diegonava6 replied:
The file name should be 'Task' not 'task'

The convention for components names is to be capitalized

diegonava6 commented on src/taskForm.jsx:
```diff
@@ -0,0 +1,17 @@
+      <input
+        className="task-input"
+        type="text"
+        placeholder="Add new task "
```
You have a space at the end of the placeholder, I mean, you have:
"Add new task "
instead of
"Add new task"

diegonava6 commented on src/task.jsx:
```diff
@@ -0,0 +1,16 @@
+    <div className={finish ? "task-conteiner finish" : "task-container "}>
+      <div className="task-text">{text}</div>
+      <div className="task-container-icons">
+        {" "}
```
Is there any reason of this space here?
{" "}

diegonava6 reviewed (commented):
Another couple of things to mention:
- Use better titles for the PR's
- Use a description, mentioning in a concise and short way what changes you did
- If is the case, use a small video with recordit  to show the changes, also you can use Screenshots on the description, depending of the case
- Use better names for the branch Diego Nava

diegonava6 commented:
One last thing is to keep in mind, is to send small PR's for the next time

Commit:
added components for tasks

Commit:
added components for tasks

diegonava6 committed:
fix changes of jose

- some changes were lost
- it is necessary to put more attention while doing merges
- keep your work/folders organized
- it was the best intent to fix the files

Commit:
hooks added

Commit:
add components to delete tasks and add tasks

Commit:
Co-authored-by: Diego Nava <diegomorenonava@gmail.com>
